app/templates/spin.hbs

```diff
@@ -0,0 +1 @@
+{{component currentSpinStage spinComplete="spinComplete" questions=model spinScore=spinScore}}
```

Comment:
This dynamically renders a component, `currentSpinStage`. Whatever value is supplied to that variable will be rendered on the page, currently there is `spin-activity` and `spin-complete`. The attributes are action and data bindings that will be available to the component.